portfolio/src/main/webapp/staticContent.js

```diff
@@ -0,0 +1,83 @@
+/** This file handles all the static UI elements of the portfolio. */
+
+/**
+ * Adds a list of hobbies to the page by displaying the hobby's description text
+ * and updating the wrapper's background color so that it appears 
+ */
+function addHobby() {
+  const hobbies = 'playing the Clarinet, reading mystery novels, (trying to) speak Spanish';
+  
+  const hobbyContainer = document.getElementById('hobby-container');
+  hobbyContainer.innerText = hobbies;
+  
+  const hobbyWrapper = document.getElementById('hobby-wrapper');
+  hobbyWrapper.style.backgroundColor = 'antiquewhite';
+}
+
+/**
+ * Adds description for each project in project section. 
+ * Input: string -- name of the selected project
```

Comment:
Consider using @param and @return

Reference:
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Reply:
Updated.